collectors/apps.plugin/apps_plugin.c

```diff
@@ -3806,6 +3849,29 @@ static void send_charts_updates_to_netdata(struct target *root, const char *type
     }
 }
 
+static void send_proc_states_count(usec_t dt)
+{
+    static bool chart_added = false;
+    // create chart for count of processes in different states
+    if (!chart_added) {
+        fprintf(
+                stdout,
+                "CHART system.process_states '' 'Apps Process States' 'processes' processes system.process_states line %d %d\n",
```

Comment:
```suggestion
                "CHART system.processes_state '' 'System Processes State' 'processes' processes system.processes_state line %d %d\n",
```

nitpick: I think "processes_state" as the metric name looks better than "process_states".

Reply:
probably this needs change to dashboard.js as well

Reply:
Yes, you are right.